hack/install-router.sh

```diff
@@ -1,34 +1,71 @@
 #!/bin/bash
```

Comment:
Hmmm, yes, this has grown complicated with the introduction of the secure master.  Needs to be easier.

@liggitt thanks for putting this together!  I'll give it a full read through and test in the morning.  Nothing jumps out at me with a quick glance.